packages/components/psammead-useful-links/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,111 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Multiple useful links should render correctly 1`] = `
+.c2 {
+  color: #222222;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.c2:hover,
+.c2:focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+  cursor: pointer;
```

Comment:
Is this needed?

Reply:
Actually, works without it. Fixed it now, thank you!